src/clib/lib/config/config_content.cpp

```diff
@@ -355,3 +323,8 @@ void config_content_pop_path_stack(config_content_type *content) {
 stringlist_type *config_content_alloc_keys(const config_content_type *content) {
     return hash_alloc_stringlist(content->items);
 }
+
+ERT_CLIB_SUBMODULE("config_content", m) {
+    m.def("get_errors",
+          [](Cwrap<config_content_type> self) { return self->parse_errors; });
```

Comment:
yeah - i don't understand this in detail, but as far as I gathered from Andrea's explanations and googling a bit, it get's initialized automagically because `std::vector` and because the struct that owns it `config_content_type` is inited with `new`